torchgen/gen_schema_utils.py

```diff
@@ -0,0 +1,90 @@
+from typing import Any, Optional, Tuple, Union
+
+from torchgen.model import (
+    Annotation,
+    Argument,
+    Arguments,
+    BaseOperatorName,
+    BaseTy,
+    BaseType,
+    CustomClassType,
+    FunctionSchema,
+    ListType,
+    OperatorName,
+    Return,
+)
+
+
```

Comment:
Put a docstring here that says something like "this isn't actually used in torchgen, it's some utilities from generating a schema from real arguments"